charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 9.4.3
+version: 9.4.4
```

Comment:
You'll need to bump the version again since https://github.com/prometheus-community/helm-charts/pull/109 was just merged